psi4/src/export_wavefunction.cc

```diff
@@ -415,8 +417,46 @@ void export_wavefunction(py::module& m) {
         .def("set_orbitals", &detci::CIWavefunction::set_orbitals, "docstring")
         .def("form_opdm", &detci::CIWavefunction::form_opdm, "docstring")
         .def("form_tpdm", &detci::CIWavefunction::form_tpdm, "docstring")
-        .def("get_opdm", &detci::CIWavefunction::get_opdm, "docstring")
-        .def("get_tpdm", &detci::CIWavefunction::get_tpdm, "docstring")
+        .def("get_opdm", &detci::CIWavefunction::get_opdm, R"pbdoc(
+              Returns the one-particle density or transition matrix.
+
+              Parameters
+              ----------
+              Iroot : int
+                  The index of the root in the bra. If Iroot and Jroot are -1, return the density matrix of root 0.
+                  Always use -1 for single-state calculations.
+              Jroot : int
+                  The index of the root in the ket. Select -1 for the same as Iroot.
+                  Always use -1 for single-state calculations.
+              spin : {'A', 'B', 'SUM'}
+                  Return the alpha density matrix, the beta density matrix, or their sum?
+              full_space : bool
+                  Return a density matrix in the space of all orbitals (true) or the active orbitals (false)?
+
+              Returns
+              -------
+              Matrix
+                  The selected one-particle density/transition matrix with Pitzer-ordered orbitals.
+                  Irrep h of the matrix corresponds to orbitals of irrep h.
+                  Element pq is <ψ|a^p a_q|ψ>.
```

Comment:
sphinx used to throw a fit if it got non-ascii. but if that hasn't changed, I can patch it up in upcoming docs PR.

Reply:
I didn't actually bother to check... If you'll handle this during your docs PR, great.